internal/org/model/org.go

```diff
@@ -0,0 +1,35 @@
+package model
+
+import (
+	es_models "github.com/caos/zitadel/internal/eventstore/models"
+	in_model "github.com/caos/zitadel/internal/model"
+)
+
+type Org struct {
+	es_models.ObjectRoot
+
+	State OrgState
+	Name  string
+}
+
+type OrgState in_model.Enum
+
+var states = []string{"Active", "Inactive"}
```

Comment:
in my opinion enum type is useless (in project and user I removed it)

Reply:
deleted because wrong PR